tests/README.md

```diff
@@ -0,0 +1,72 @@
+**Warning:** the test suite runs _invasively_ against the Kubernetes cluster that your local `kubectl` is currently configured against.
+
+## Usage
+
+Invoke `make bats-tests` in the root of this repository.
+
+
+
+### Test local dev state (artifacts not pushed)
+
+Not yet supported.
+Let's change this ASAP.
+
+This test suite for now assumes public availability of a Helm chart on GHCR or NGC, pointing to a container image publicly available on GHCR or NGC.
+
+### Test Helm chart from registery
+
+#### Default versions
+
+Say, this is the current local git revision:
+
+```console
+$ git rev-parse --short=8 HEAD
+e6e1dde4
+```
+
+Then the test suite runs with the default configuration, for example:
+```console
+$ make bats-tests
```

Comment:
```suggestion
$ make -f tests/bats/Makefile
```

Reply:
Moved files/logic down to `tests/bats`, added a top-level target: `make bats`.